plot/plot-config.js

```diff
@@ -1,311 +1,370 @@
 var plotConfigs = {
```

Comment:
you should remove this `plot-config.js` file as well (like the other config files)